package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "courseography",
+  "version": "0.2.0.0",
+  "main": "index.js",
```

Comment:
This can probably be removed since there's no need for a Node module entry point.